examples/create_table_cloud.ts

```diff
@@ -0,0 +1,27 @@
+import { createClient } from '@clickhouse/client'
+
+// ClickHouse cloud - a specific CREATE TABLE syntax is allowed
```

Comment:
What is the purpose of `a specific CREATE TABLE syntax is allowed` comment?

Reply:
Removed